driver/src/main.rs

```diff
@@ -260,6 +260,16 @@ struct Options {
         use_delimiter = true
     )]
     gas_estimators: Vec<GasEstimatorType>,
+
+    /// Whether to rely on external price sources (e.g. 1Inch, Kraken etc)
+    /// when estimating token prices
+    #[structopt(
+        long,
+        env = "USE_SOLUTION_SUBMITTER",
+        parse(try_from_str),
```

Comment:
Is try_from_str needed?

Reply:
Yes, because otherwise structopt will take the sheer existence of a `USE_SOLUTION_SUBMITTER` env variable as a true. In our kubernetes env we need to declare the variable unconditionally and rely on setting it to different values (e.g. true/false or 1/0) in different environments. For this, we need to parse from string.